src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -3122,7 +3122,10 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST_TL(GUITest_common_scenarios_NIAID_pipelines::test_0005, minutes(7), labels({Cistrome, Linux, Windows, MacOS}));
     REGISTER_TEST_TL(GUITest_common_scenarios_NIAID_pipelines::test_0006, minutes(7), labels({Cistrome, Linux, Windows, MacOS}));
     REGISTER_TEST(GUITest_common_scenarios_NIAID_pipelines::test_0007);
-
+    REGISTER_TEST_L(GUITest_common_scenarios_NIAID_pipelines::test_0008, labels({Cistrome, Linux, Windows, MacOS}));
```

Comment:
What is the timeout for these tests? 
If these tests are fast enough (<4mins) and can be run on any Linux/Windows/MacOS they should be in the Nightly suite: see test_0007 registration on the line above.